cdap-app-fabric/src/test/java/io/cdap/cdap/support/services/SupportBundleServiceTest.java

```diff
@@ -134,32 +133,21 @@ public void testDeleteOldBundle() throws Exception {
     File tempFolder = new File(configuration.get(Constants.SupportBundle.LOCAL_DATA_DIR));
     createNamespace("default");
     String path = String.format("%s/support/bundle?namespace=default", Constants.Gateway.API_VERSION_3);
-    List<String> bundleIdList = new ArrayList<>();
-    for (int i = 0; i < 7; i++) {
+    String expectedDeletedUuid = "";
```

Comment:
If I understand this correctly, this simply removes the ArrayList because it doesn't need to store all of the other UUIDs, and then you perform a minor refactor to pull the status update code out into a helper method? If my understanding is correct, then LGTM

Reply:
yeah this test to monitor if we delete the oldest bundle or failed state bundle first, because in local directory we want to only store 7 bundles per instances

Reply:
Gotcha, thanks!